### DISCUSSION TITLE
Python prompt API

### DISCUSSION THREAD

**Discussion by Code4SAFrankie** [civitai/civitai#371] [Ideas]
A Python API to retrieve the prompt examples for a model would be awesome.

**Comment by WASasquatch** [comment]
Isn't there already a REST API you can use in whatever?

**Reply by Code4SAFrankie** [reply]
Huh?

**Reply by WASasquatch** [reply]
> Huh?

https://civitai.com/api/v1/models?limit=100&sort=Newest

**Reply by Code4SAFrankie** [reply]
Thanks
